lib/audio_pool.dart

```diff
@@ -11,11 +11,11 @@ typedef void Stoppable();
 /// All AudioPlayers loaded are for the same [sound]. If you want multiple sounds use multiple [AudioPool].
 /// Use this class if you'd like have extremely quick firing, repetitive and simultaneous sounds, like shooting a laser in a fast-paced spaceship game.
 class AudioPool {
-  AudioCache cache;
+  late AudioCache cache;
```

Comment:
if we move the instantiation of this to a constructor initializer it doesn't need to be late right?

Reply:
you are correct! just updated